### PR TITLE
Change Graph.InstallEsx to Graph.InstallESXi

### PR DESCRIPTION
**Pull request by anhou**
resolve RackHD/RackHD#218

@RackHD/corecommitters

### CODE REVIEW

File: lib/api/redfish-1.0/systems.js [hunk 1]
```diff
@@ -579,7 +579,7 @@ var doBootImage = controller(function(req,res) {
             graphName = 'Graph.InstallCentOS';
             graphOptions.defaults.osName = 'CentOS';
         } else if(payload.osName.indexOf('ESXi') !== -1) {
-            graphName = 'Graph.InstallEsx';
+            graphName = 'Graph.InstallESXi';
             graphOptions.defaults.osName = 'Esx';
```
Comment (yyscamper) [inline lib/api/redfish-1.0/systems.js]: Why not change this one?

**Comment by yyscamper** [comment]
jenkins: test this please

**Comment by iceiilin** [comment]
:+1:

**Comment by yyscamper** [comment]
:+1:

File: lib/api/redfish-1.0/systems.js (hunk 1 above)
Comment (anhou) [inline lib/api/redfish-1.0/systems.js]: this is related to functions in codes, if changed, there will be function names and variables changed, avoid here, just change interface

**Comment by anhou** [comment]
test this please

**Pull request merged**